src/systemctl/systemctl.c

```diff
@@ -8220,7 +8220,10 @@ static int talk_initctl(void) {
 
         request.runlevel = rl;
 
-        fd = open(INIT_FIFO, O_WRONLY|O_NONBLOCK|O_CLOEXEC|O_NOCTTY);
+        fd = open("/run/initctl", O_WRONLY|O_NONBLOCK|O_CLOEXEC|O_NOCTTY);
```

Comment:
There are some references to INIT_FIFO a few lines down, mostly on log_error_errno, but I think you probably should keep track of that otherwise it might get confusing, trying to open one file and logging about failure to open the other one...

Reply:
This doesn't seem like a big deal to me; I think I will wait for additional feedback before over-designing the error message handling here.